### PR TITLE
doc: add example of setting pool in cephfs layout

### PR DESCRIPTION
**Pull request by jcsp**
Signed-off-by: John Spray <john.spray@redhat.com>

### CODE REVIEW

File: doc/cephfs/file-layouts.rst [hunk 1]
```diff
@@ -201,15 +201,27 @@ directories do not have layouts set:
...
     .... data pools: [cephfs_data cephfs_data_ssd ]
 
 Make sure that your cephx keys allows the client to access this new pool.
+
+You can then update the layout on a directory in CephFS to use the pool you added:
+
+.. code-block:: bash
+
+    $ mkdir /mnt/cephfs/myecdir
+    $ setfattr -n ceph.dir.layout.pool -v cephfs_data_ssd /mnt/cephfs/myecdir
```
Comment (dillaman) [inline doc/cephfs/file-layouts.rst]: Unrelated to this doc change -- but do we need a process to auto-tag the data pool as being owned by CephFS when this attribute is set?

Comment (jcsp) [reply doc/cephfs/file-layouts.rst]: Nope, somebody called "Jason" already updated add_data_pool's implementation to add the tag.  Smart guy!

Comment (dillaman) [reply doc/cephfs/file-layouts.rst]: Thank goodness -- just wanted to ensure there was no magic backdoor of config settings.

Comment (Millnert) [reply doc/cephfs/file-layouts.rst]: I think you should reconcile the original doc using ``cephfs_data_ssd`` pool name with your intent here to document EC pools via the ``myecdir``folder name.
Pick one, or duplicate for both?

Comment (jcsp) [reply doc/cephfs/file-layouts.rst]: Fair point, head was in two places.  Updated.

File: doc/cephfs/file-layouts.rst [hunk 2]
```diff
@@ -201,15 +201,27 @@ directories do not have layouts set:
...
+You can then update the layout on a directory in CephFS to use the pool you added:
+
+.. code-block:: bash
+
+    $ mkdir /mnt/cephfs/myssddir
+    $ setfattr -n ceph.dir.layout.pool -v cephfs_data_ssd /mnt/cephfs/myssddir
+
+All new files created within that directory will now inherit its layout and place their data in your newly added pool. 
+
+You may notice that object counts in your primary data pool (the one passed to `fs new`) continue to increase, even if files are being created in the pool you added.  This is normal: the file data is stored in the pool specified by the layout, but a small amount of metadata is kept in the primary data pool for all files.
```
Comment (alfredodeza) [inline doc/cephfs/file-layouts.rst]: if "fs new" is meant to be displayed in a mono-spaced font then double backticks are needed (vs. single ones)

Comment (jcsp) [reply doc/cephfs/file-layouts.rst]: Done

**Pull request merged**